'TestHTTP2NetworkXX' failed; it was caused by empty response

```
--- FAIL: TestHTTP2NetworkSendMessage (0.12s)
	http2_network_test.go:206: 
			Error Trace:	http2_network_test.go:206
			Error:      	Not equal: 
			            	expected: ""
			            	actual  : "{\"T\":\"dummy-message\",\"Hash\":\"5sD1FhUsNwHzFnHQVQa6fm7H8Rhuz8M7CXtSondscTVc\",\"Data\":\"findme\"}"
			            	
			            	Diff:
			            	--- Expected
			            	+++ Actual
			            	@@ -1 +1 @@
			            	-
			            	+{"T":"dummy-message","Hash":"5sD1FhUsNwHzFnHQVQa6fm7H8Rhuz8M7CXtSondscTVc","Data":"findme"}
			Test:       	TestHTTP2NetworkSendMessage
			Messages:   	The sendMessage and the return should be the same.
127.0.0.1 - - [27/Jun/2018:09:42:23 +0000] "POST /node/ballot HTTP/2.0" 200 91 "" "v-n1"
FAIL
FAIL	github.com/spikeekips/sebak/lib/network	34.382s
```

--- a/lib/network/http2_network_test.go
+++ b/lib/network/http2_network_test.go
@@ -67,6 +67,7 @@ const (
 	keyPath  = "key.pem"
 )
 
+// Waiting until the server is ready
 func pingAndWait(t *testing.T, c0 NetworkClient) {
 	waitCount := 0
 	for {

--- a/lib/network/http2_network_test.go
+++ b/lib/network/http2_network_test.go
@@ -67,6 +67,21 @@ const (
 	keyPath  = "key.pem"
 )
 
+func pingAndWait(t *testing.T, c0 NetworkClient) {
+	waitCount := 0
+	for {
+		if b, err := c0.GetNodeInfo(); len(b) != 0 && err == nil {
+			break
+		} else {
+			time.Sleep(time.Millisecond * 100)
+			waitCount++
+			if waitCount > 100 {
+				t.Error("Server is not available")
+			}
+		}
+	}
+}
+
 func createNewHTTP2Network(t *testing.T) (kp *keypair.Full, mn *HTTP2Network, validator *sebakcommon.Validator) {
 	g := NewKeyGenerator(dirPath, certPath, keyPath)
 
@@ -162,6 +177,7 @@ func TestHTTP2NetworkMessageBrokerResponseMessage(t *testing.T) {
 	defer s0.Stop()
 
 	c0 := s0.GetClient(s0.Endpoint())
+	pingAndWait(t, c0)
 
 	returnMsg, _ := c0.Connect(currentNode)
 
@@ -177,6 +193,7 @@ func TestHTTP2NetworkConnect(t *testing.T) {
 	defer s0.Stop()
 
 	c0 := s0.GetClient(s0.Endpoint())
+	pingAndWait(t, c0)
 
 	o, _ := currentNode.Serialize()
 	nodeStr := removeWhiteSpaces(string(o))
@@ -196,6 +213,7 @@ func TestHTTP2NetworkSendMessage(t *testing.T) {
 	defer s0.Stop()
 
 	c0 := s0.GetClient(s0.Endpoint())
+	pingAndWait(t, c0)
 
 	msg := NewDummyMessage("findme")
 	returnMsg, _ := c0.SendMessage(msg)
@@ -214,6 +232,7 @@ func TestHTTP2NetworkSendBallot(t *testing.T) {
 	defer s0.Stop()
 
 	c0 := s0.GetClient(s0.Endpoint())
+	pingAndWait(t, c0)
 
 	msg := NewDummyMessage("findme")
 	returnMsg, _ := c0.SendBallot(msg)

